awx/ui_next/src/screens/Job/JobList/JobList.test.jsx

```diff
@@ -87,93 +87,161 @@ UnifiedJobsAPI.read.mockResolvedValue({
   data: { count: 3, results: mockResults },
 });
 
-describe('<JobList />', () => {
-  test('initially renders succesfully', async done => {
-    const wrapper = mountWithContexts(<JobList />);
-    await waitForElement(
-      wrapper,
-      'JobList',
-      el => el.state('jobs').length === 6
-    );
+function waitForLoaded(wrapper) {
+  return waitForElement(
+    wrapper,
+    'JobList',
+    el => el.find('ContentLoading').length === 0
+  );
+}
 
-    done();
+describe('<JobList />', () => {
+  test('initially renders succesfully', async () => {
+    let wrapper;
+    await act(async () => {
+      wrapper = mountWithContexts(<JobList />);
+    });
+    await waitForLoaded(wrapper);
```

Comment:
You call this in each test.  Why not just do this inside of a `beforeEach || beforeAll`?